sevenn/pair_e3gnn/README.md

```diff
@@ -0,0 +1,133 @@
+
+```bash
+sevenn_patch_lammps ./lammps_sevenn --d3
+```
+
+You can follow the remaining installation steps in the SevenNet documentation. For detailed installation options, refer to `sevenn/pair_e3gnn/patch_lammps.sh`.
+
+**PLEASE NOTE**: In this code, the selection of compute capability for GPUs relies on FindCUDA in libtorch. However, the default CUDA architectures selected in CUDA 12.1 are `"50;80;86;89;90"`. The D3 code, however, requires at least compute capability 6.0. If you compile with the default settings, you may encounter an `atomicAdd` error. To resolve this issue, I have included code in the script to remove the compute capability for 5.0.
+
+If you want to manually specify the compute capability, note that libtorch does not use `CMAKE_CUDA_ARCHITECTURES` or `CUDA_ARCHITECTURE` in CMake (ignore the CMake log). Instead, you can configure it manually by setting the environment variable `TORCH_CUDA_ARCH_LIST="60;71;80;86;89;90"`, which will then be automatically inserted into the nvcc flags, as libtorch does.
+
+**PLEASE NOTE**: Setting `fmad=false` for the NVCC compiler is essential to obtain precise results. This is the default option as described in patch_lammps.sh.
+
+# Usage for LAMMPS
+
+You can use the D3 dispersion correction in LAMMPS with SevenNet by using the `pair/hybrid` command:
+
+```txt
+pair_style pair/hybrid e3gnn d3 {cutoff_d3} {cutoff_d3_CN} {type_of_damping}
+pair_coeff e3gnn * * {path_to_serial_model} {space_separated_chemical_species}
+pair_coeff d3 * * {path_of_r0ab.csv} {path_of_d3_pars.csv} {name_of_functional} {space_separated_chemical_species}
+compute {name_of_your_compute} all pressure NULL virial pair/hybrid
+atom_modify sort 0 0 # Adhoc solution for the issue on 3Aug2023, which will be resolved soon
+```
+
+for example,
+
+```txt
+pair_style pair/hybrid e3gnn d3 9000 1600 d3_damp_bj
+pair_coeff e3gnn * * ./deployed_serial.pt C H O
+pair_coeff d3 * * ./r0ab.csv ./d3_pars.csv pbe C H O
+compute vp_d3 all pressure NULL virial pair/hybrid d3
+atom_modify sort 0 0
+```
+
+You can find `r0ab.csv` and `d3_pars.csv` files in the `pair_e3gnn` directory. These files are necessary to calculate D3 interactions.
+
+`cutoff_d3` and `cutoff_d3_CN` are *square* of cutoff radii for energy/force and coordination number, respectively. Units are Bohr radius: 1 (Bohr radius) = 0.52917721 (Å). Default values are `9000` and `1600`, respectively. this is also the default values used in VASP. check the [note](#note-the-default-cutoff-parameters) for more informaiton.
+
+Available `type_of_damping` are as follows:
+- `d3_damp_zero`: Zero damping
+- `d3_damp_bj`: Becke-Johnson damping
+
+`name_of_functional` should match the exchange-correlation functional of the DFT calculations in the training set for the SevenNet potentials. For instance, SevenNet-0 was trained using the PBE functional, so you should specify `pbe` when using it.
```

Comment:
Supporting exchange-correlation functionals: ~~
Note that SevenNet-0 is trained on the 'PBE' functional